lib/matplotlib/_mathtext_data.py

```diff
@@ -1056,11 +1056,16 @@
             (0x0053, 0x0059, 'rm', 0x1d54a), # S-Y
             (0x005a, 0x005a, 'rm', 0x2124),  # Z
             (0x0061, 0x007a, 'rm', 0x1d552), # a-z
-            (0x0393, 0x0393, 'rm', 0x213e),  # \Gamma
-            (0x03a0, 0x03a0, 'rm', 0x213f),  # \Pi
-            (0x03a3, 0x03a3, 'rm', 0x2140),  # \Sigma
-            (0x03b3, 0x03b3, 'rm', 0x213d),  # \gamma
-            (0x03c0, 0x03c0, 'rm', 0x213c),  # \pi
+            (ord("\N{GREEK CAPITAL LETTER GAMMA}"),
```

Comment:
I would also replace the codepoints above (0x0030 = `"0"` (no real need to write that as `"\N{DIGIT ZERO}"`)), as well as the last items of the tuple *when applicable* (0x213e = `"\N{DOUBLE-STRUCK CAPITAL GAMMA}"` but 0x1d7d8 is just an internal codepoint for the font with no associated name).

Reply:
Hi, thanks for the feedback! I included replacements for these codepoints and for the last items that i could find as well in my last commit